app/routes/circulars/circulars.lib.ts

```diff
@@ -76,6 +76,25 @@ export function formatDateISO(date: number) {
   return new Date(date).toISOString().replace(/\.\d+Z$/, 'Z')
 }
 
+/** convert a ms value to mm-dd-yyy format */
+export function formatDateMMDDYYYY(date: number) {
+  const dateObj = new Date(date)
+  var dateStr = dateObj.toLocaleDateString('en-US', {
+    year: 'numeric',
+    month: '2-digit',
+    day: '2-digit',
+  })
+  return dateStr.replace(/\//g, '-') // replace / with -
+}
+
+/** convert a date in format mm-dd-yyyy to ms since 01/01/1970 */
+export function formatDateEpoch(date: string | number) {
+  if (typeof date === 'string') {
+    return new Date(date).getTime()
+  }
+  return date
+}
```

Comment:
Use the format `YYYY-MM-DD`, not `MM-DD-YYYY`, to more closely match the international ISO 8601 format.

Reply:
I added a function for YYYY-MM-DD in 0e2c267. Given the goal to have the input and url string be human readable in  iso8601, this function might end up being unneeded, as the start and end dates should just be passed as strings at all points and only converted to unix time inside the search function. That being said, looking around a bit more, the default infinity values don't seem to interact well when using the Date objects. One solution to that could be to change it so the default start and end dates are undefined and at the point of conversion inside the search, they are set to infinity and negative infinity if they're undefined (similar to how query is defined when there is no query), though this would require tweaking the declaration of their types in the various functions.

Reply:
Right. We don't need the function to convert from epoch to ISO. We only need the function to convert from ISO to epoch.

I agree: use undefined, not infinity.

Reply:
> Right. We don't need the function to convert from epoch to ISO. We only need the function to convert from ISO to epoch.
> 
> I agree: use undefined, not infinity.

made the change so they're undefined in 64a34b2.